Fix code review issues

ayeleta opened the pull request:
_GetRootResources - actually find the parents
Don't cache root resources
activate routes: don't use activatetopology
pep 8 method names

<!-- Reviewable:start -->

---

This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/qualisystemslab/awesome-sandbox/6)

<!-- Reviewable:end -->

ayeleta committed:
Fix code review issues

_GetRootResources - actually find the parents
Don't cache root resources
activate routes: don't use activatetopology
pep 8 method names

AlonSaggie commented:
Review status: 0 of 5 files reviewed at latest revision, 1 unresolved discussion.

---

_[QualiEnvironmentUtils/Networking/NetworkingSaveNRestore.py, line 22 [r1]](https://reviewable.io:443/reviews/qualisystemslab/awesome-sandbox/6#-KIZqKYQeui0O7w1yRbv:-KIZqKYQeui0O7w1yRbw:-425803652) ([raw file](https://github.com/qualisystemslab/awesome-sandbox/blob/d7b6a246446ecda1d6fd52230f1477029de27c57/QualiEnvironmentUtils/Networking/NetworkingSaveNRestore.py#L22)):_

> ``` Python
>                         self.config_files_root = 'tftp://' + resource.address + "/" + tftp_server_destination_path
>                         break
>         if tftp_server_destination_path =="":
> ```

Looks like you miss a line below...
don't forget to set the 'writetooutput' as False

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qualisystemslab/awesome-sandbox/6)_

<!-- Sent from Reviewable.io -->

AlonSaggie commented:
Reviewed 5 of 5 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qualisystemslab/awesome-sandbox/6)_

<!-- Sent from Reviewable.io -->

ayeleta committed:
Fix code review issues

_GetRootResources - actually find the parents

Don't cache root resources

activate routes: don't use activatetopology
pep 8 method names

AlonSaggie commented:
Reviewed 4 of 4 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_[QualiEnvironmentUtils/Sandbox.py, line 77 [r2]](https://reviewable.io:443/reviews/qualisystemslab/awesome-sandbox/6#-KIZwCLKPsiCPCy-KrJn:-KIZwCLKPsiCPCy-KrJo:1996141163) ([raw file](https://github.com/qualisystemslab/awesome-sandbox/blob/1d4942b3d12d5939434da05f3e988309484d402a/QualiEnvironmentUtils/Sandbox.py#L77)):_

> ``` Python
>             resource_path = ''
>             if resource.FolderFullPath:
>                 resource_path = resource.FolderFullPath + "/"
> ```

No need for full path - resource names are unique...

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qualisystemslab/awesome-sandbox/6)_

<!-- Sent from Reviewable.io -->
